VCProject.Base/VCHelper.cs

```diff
@@ -41,6 +41,12 @@ private VCFileConfiguration GetVCFileConfig(Document document, out string reason
                 return null;
             }
 
+            if (vcFile.FileType == eFileType.eFileTypeCppHeader)
+            {
+                reasonForFailure = ""; // Empty string == skip output for it.
+                return null;
+            }
+
             if (vcFile.FileType != eFileType.eFileTypeCppCode && vcFile.FileType == eFileType.eFileTypeCppClass)
```

Comment:
Correction: it tries to compile the .h file. All is well.  Still a bit of a waste though.